client/components/info-popover/index.jsx

```diff
@@ -83,6 +92,42 @@ export default class InfoPopover extends Component {
 		}
 	};
 
+	handleOnMouseEnterButton = () => {
+		const { onOpen, showOnHover } = this.props;
+
+		if ( ! showOnHover ) {
+			return;
+		}
+
+		onOpen?.();
+		this.setState( { showPopover: true }, this.recordStats );
+	};
+
+	handleOnMouseLeave = () => {
+		setTimeout( () => {
+			const { showOnHover } = this.props;
+
+			if ( ! showOnHover ) {
+				return;
+			}
+
+			if ( this.inPopover ) {
+				return;
+			}
+
+			this.setState( { showPopover: false }, this.recordStats );
+		}, 100 );
```

Comment:
Could you try bumping this up a bit and seeing if it still feel okay? It would help prevent the popover from auto-dismissing when the user accidentally moves the cursor briefly outside of the info button.

Reply:
250ms seems to be the sweet spot, thanks for the review @mattsherman

Reply:
This feels much better!